cookbooks/retention.md

```diff
@@ -0,0 +1,214 @@
+*Authored by the Product Data Science Team. Please direct questions/concerns to Ben Miroglio (`bmiroglio`).*
+
+# Retention
+
+Retention measures the rate at which users are *continuing* to use Firefox, making it one of the more important metrics we track. We commonly measure retention between releases, experiment cohorts, and various Firefox subpopulations to better understand how a change to the user experience or use of a specific feature affect behavior.
+
+### N Week Retention
```

Comment:
good work! One more distinction I would make about types of N week retention: 

**rolling vs cohorted**

Here you're using a rolling approach to N week retention (N week is between (7*n) and ((7*n)+ 6) after client's pcd date), but a lot of people are using cohorts to calculate retention: 

Define weekly cohorts for your population (the clients that start between 20180101 - 20180107 = cohort1, 201801018- 20180114 = cohort2, etc.). For each cohort, define weekN retention as, appeared in the week period N weeks after the cohort period (i.e. week1 is appeared in 201801018- 20180114 for cohort 1)

So essentially, using an anchor **period** instead of an anchor **point**. 

Would be a good idea to clarify this distinction, since this is many people's understanding of N week retention right now.

Reply:
Good point. 

I use cohorts when there isn't a clear anchor point (i.e. users that enabled sync--there is no clear *sync was enabled on this date* indicator without looking across a client's history). Might be worth adding an example like this as well in section title "What if you don't have an achor point?". 

When you do have an anchor point however, IMO the rolling method is the most precise (and least expensive).